app/services/yelp_service.rb

```diff
@@ -0,0 +1,24 @@
+class YelpService
+  def initialize(activity,latitude, longitude)
+    @activity = activity
+    @latitude = latitude
+    @longitude = longitude
+  end
+
+  def self.get_restaurants(activity, latitude_value, longitude_value)
+    new(activity, latitude_value, longitude_value).get_restaurants
+  end
```

Comment:
100 % sure how the instructors are going to feel about this but I'm good with it.